openedx_ledger/models.py

```diff
@@ -153,6 +153,8 @@ def balance(self, committed_only=False):
         """
         Calculate the current balance of the ledger.
 
+        TODO: propagate committed_only down into the reversals too.
```

Comment:
This seems logical to me, but out of scope for this particular PR so I dropped a comment instead.